src/matchers/__tests__/to-have-accessible-name.test.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { View, Text, TextInput } from 'react-native';
+import { render, screen } from '../..';
+import '../extend-expect';
+
+test('toHaveAccessibleName() on view with accessibilityLabel prop', () => {
+  render(<View testID="accessibility-label" accessibilityLabel="Test Label" />);
+  const element = screen.getByTestId('accessibility-label');
+  expect(element).toHaveAccessibleName('Test Label');
```

Comment:
Also check if `.toHavaAccessibleName()` (without name)-  it should verify if the name is present regardles of actual value.

Add also negative cases, e.g. `not.toHaveAccessibleName("Other Label")